### PR TITLE
fix #458 : fix description of TODO

### LINKED ISSUE #458: Confusion in Nginx, #414
Todo #345 demands handling `*.main.config` files but as an example gives `"www.example.com.hosts.conf"`. I think I think should be `"www.example.com.main.conf"`, instead. Please, fix it.

Next, the method `updateConfigScript` updates `*.hosts.config` files representing Load Balancing Group. I think it should rather be called `updateLoadBalancingGroupScript` or similar. Please, fix it.

Then the Todo #345 becomes clear that requires creation of a method `updateServerConfigScript` which should handle `*.main.config` files.

Please, correct me if I am wrong.

### PR DESCRIPTION
**Pull request by wb14123**
This patch fix #458

### CODE REVIEW

**Comment by davvd** [comment]
Thanks for the pull request, let me find a reviewer..

**Comment by davvd** [comment]
@krzyk please go ahead with this task,it's yours

File: src/main/java/com/thindeck/steps/Nginx.java [hunk 1]
```diff
@@ -136,7 +136,9 @@ public void update(@NotNull final String host, @NotNull final int hport,
      * @param sport Server port to redirect requests to
      * @return Commands for updating hosts configuration.
      */
-    private String updateConfigScript(final String host, final String server,
+    private String updateLoadBalancingScript(
```
Comment (krzyk) [inline src/main/java/com/thindeck/steps/Nginx.java]: I think that right now we this method creates a script that updates load balancer, and it implements interface so both `update` and `LoadBalancing` are not required. So I would leave just `script(...)`

File: src/main/java/com/thindeck/steps/Nginx.java [hunk 2]
```diff
@@ -136,7 +136,9 @@ public void update(@NotNull final String host, @NotNull final int hport,
...
+        final String host,
```
Comment (krzyk) [inline src/main/java/com/thindeck/steps/Nginx.java]: If the params fit in line they should be in the same as the method name (80 chars per line)

**Comment by krzyk** [comment]
@wb14123 two comments above

**Comment by wb14123** [comment]
@krzyk  The `Nginx` class will have to do two things in order to update the load balancin:

1. Update the load balancing script, which is *.hosts.conf.
2. Update the server config script, which is *.main.conf.

So `update` will invoke two methods, one of them is `updateLoadBlancingScript` as I wrote, and another one should be a method like `updateMainScript` which will be written in #414 .

If I'm right I will just fix the code style in the next commit.

**Comment by krzyk** [comment]
@wb14123 we try to keep shorter method (variables and fields also) names in this project. For sure `update` part is not required, the whole purpose of this class is to do an update of load balancer configuration, so there is no need to write that over and over.

So that leaves `loadBalancingScript`, rename the method to that (and move parameters as they were before) and let's see if @yegor256 is OK with that, during the merge.

**Comment by wb14123** [comment]
@krzyk  Done.

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor merge

**Comment by krzyk** [comment]
@yegor256 please try again, some DNS issue

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by davvd** [comment]
@krzyk Much appreciated! **18 mins** was added to your account, payment ID `50000677`... +18 to your rating, your total score is [+1360](http://www.netbout.com/b/35035?open=rating)

**Comment by davvd** [comment]
@rultor deploy pls

**Pull request merged**